Content.Shared/Turnstile/Systems/TurnstileSystem.cs

```diff
@@ -31,19 +31,20 @@ private void OnComponentStartup(Entity<TurnstileComponent> ent, ref ComponentSta
 
     private void CreateSpinner(Entity<TurnstileComponent, TransformComponent?> ent)
     {
-        if (!Resolve<TransformComponent>(ent, ref ent.Comp2))
+        if (!Resolve(ent, ref ent.Comp2))
             return;
 
-        if (ent.Comp1.SpinnerUid != EntityUid.Invalid)
+        if (ent.Comp1.SpinnerUid != null && ent.Comp1.SpinnerUid != EntityUid.Invalid)
```

Comment:
no need to check for invalid just check for null

Reply:
It seems to always be 0, rather than to null, when the object gets spawned. How can I prevent that behaviour?

Reply:
EntityUid? defaults to null not Invalid